File: fizzbuzz.c
#include <stdio.h>
#include <string.h>

int i;
int usr;
char one[50];
char two[50];
char fixer[50];

int main(){
    //add input
    printf("Give me a number :");
    scanf("%d", &usr);
    printf("Give me a short word NOW!!!!!!");
    scanf("%s", one);
    printf("Give me a another short word :)");
    scanf("%s", two);
    strcat(fixer, one);
    strcat(fixer, two);
    //loop that counts to input
    for(i=1;i<=usr;i++){
        //replace #'s divisible by 3 and 5 with input
        if(i%3==0 && i%5==0){
            printf("%s\n", fixer);
        //replace #'s divisible by 3 with input
        }else if(i%3==0){
            printf("%s\n", one);
        //replace #'s divisible by 5 with input
        }else if(i%5==0){
            printf("%s\n", two);
    //print the number
        }else{
            printf("%d \n", i);
        }
    }
}

File: fizzbuzz.py
#ask for a number, word1 and word2
i = 1
num = int(input("Give me a number. \n"))
fword = (input("Give me a word.\n"))
sword = (input("Please give me another word. \n"))
#replace numbers divisible by 5 and 3 with input1 + input2
while i <= num:
    if i%3==0 and i%5==0:
        print(fword + sword)
    
#replace numbers divisible by 3 with input1
    elif i%3==0:
        print(fword)
#replace numbers divisible by 5 with input2
    elif i%5==0:
        print(sword)
#count to input
    else:
        print(i)
    i+=1